api/tests/certification/session/unit/domain/usecases/upload-cpf-files_test.js

```diff
@@ -15,7 +15,7 @@ describe('Unit | UseCase | upload-cpf-files', function () {
   context('#upload', function () {
     it('should instantiate an Upload with the expected parameters', async function () {
       // given
-      const startUploadStub = sinon.stub(S3ObjectStorageProvider.prototype, 'startUpload');
+      const startUploadStub = sinon.stub(cpfExportsStorage, 'sendFile');
```

Comment:
```suggestion
        const sendFileStub = sinon.stub(cpfExportsStorage, 'sendFile');
```
Et utiliser ce nouveau nom plus bas :)

Reply:
Bien vu, done !